src/app/API.service.ts

```diff
@@ -0,0 +1,838 @@
+            __typename
+            id
+            name
+            age
+            createdAt
+            updatedAt
+          }
+          createdAt
+          updatedAt
+          departmentManagerId
+        }
+      }`;
+    const gqlAPIServiceArguments: any = {
+      input
+    };
+    if (condition) {
+      gqlAPIServiceArguments.condition = condition;
+    }
+    const response = (await API.graphql(
+      graphqlOperation(statement, gqlAPIServiceArguments)
+    )) as any;
+    return <UpdateDepartmentMutation>response.data.updateDepartment;
+  }
+  async DeleteDepartment(
+    input: DeleteDepartmentInput,
+    condition?: ModelDepartmentConditionInput
+  ): Promise<DeleteDepartmentMutation> {
+    const statement = `mutation DeleteDepartment($input: DeleteDepartmentInput!, $condition: ModelDepartmentConditionInput) {
+        deleteDepartment(input: $input, condition: $condition) {
+          __typename
+          id
+          name
+          manager {
+            __typename
+            id
+            name
+            age
+            createdAt
+            updatedAt
+          }
+          createdAt
+          updatedAt
+          departmentManagerId
+        }
+      }`;
+    const gqlAPIServiceArguments: any = {
+      input
+    };
+    if (condition) {
+      gqlAPIServiceArguments.condition = condition;
+    }
+    const response = (await API.graphql(
+      graphqlOperation(statement, gqlAPIServiceArguments)
+    )) as any;
+    return <DeleteDepartmentMutation>response.data.deleteDepartment;
+  }
+  async CreateEmployee(
+    input: CreateEmployeeInput,
+    condition?: ModelEmployeeConditionInput
+  ): Promise<CreateEmployeeMutation> {
+    const statement = `mutation CreateEmployee($input: CreateEmployeeInput!, $condition: ModelEmployeeConditionInput) {
+        createEmployee(input: $input, condition: $condition) {
+          __typename
+          id
+          name
+          age
+          createdAt
+          updatedAt
+        }
+      }`;
+    const gqlAPIServiceArguments: any = {
+      input
+    };
+    if (condition) {
+      gqlAPIServiceArguments.condition = condition;
+    }
+    const response = (await API.graphql(
+      graphqlOperation(statement, gqlAPIServiceArguments)
+    )) as any;
+    return <CreateEmployeeMutation>response.data.createEmployee;
+  }
+  async UpdateEmployee(
+    input: UpdateEmployeeInput,
+    condition?: ModelEmployeeConditionInput
+  ): Promise<UpdateEmployeeMutation> {
+    const statement = `mutation UpdateEmployee($input: UpdateEmployeeInput!, $condition: ModelEmployeeConditionInput) {
+        updateEmployee(input: $input, condition: $condition) {
+          __typename
+          id
+          name
+          age
+          createdAt
+          updatedAt
+        }
+      }`;
+    const gqlAPIServiceArguments: any = {
+      input
+    };
+    if (condition) {
+      gqlAPIServiceArguments.condition = condition;
+    }
+    const response = (await API.graphql(
```

Comment:
What if this operation fails? Probably it's better to wrap this call into `try catch` block and handle error properly? I'm talking not only about this place, but about all places where you call `API.graphql()`.